.github/workflows/ci.yml

```diff
@@ -177,7 +163,7 @@ jobs:
           build-args: |
             MIX_ENV=test
     outputs:
-      image: ${{ steps.meta.outputs.tags }}
+      image: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME_TESTING }}@${{ steps.docker.outputs.digest }}
```

Comment:
Using the sha is safer because if the image has multiple tags (e.g., `latest` and version number) we cannot use them as a list to pull the image. Then we would need to extract only one tag.